adoptopenjdk-api-v3-persistance/src/main/kotlin/net/adoptopenjdk/api/v3/dataSources/APIDataStore.kt

```diff
@@ -69,17 +72,26 @@ object APIDataStore {
     }
 
     @VisibleForTesting
-    fun loadDataFromDb(): AdoptRepos {
+    fun loadDataFromDb(forceUpdate: Boolean): AdoptRepos {
         binaryRepos = runBlocking {
-            val data = variants
-                .versions
-                .map { version ->
-                    ApiPersistenceFactory.get().readReleaseData(version)
-                }
-                .filter { it.releases.nodes.isNotEmpty() }
-                .toList()
-
-            AdoptRepos(data)
+            val updated = ApiPersistenceFactory.get().getUpdatedAt()
+
+            if (forceUpdate || updated != updatedAt) {
+                val data = variants
+                    .versions
+                    .map { version ->
+                        ApiPersistenceFactory.get().readReleaseData(version)
+                    }
+                    .filter { it.releases.nodes.isNotEmpty() }
+                    .toList()
+                updatedAt = ApiPersistenceFactory.get().getUpdatedAt()
+
+                val newData = AdoptRepos(data)
+                showStats(binaryRepos, newData)
+                newData
```

Comment:
Not sure what that line does...

Reply:
it is the value returned from the statement